core/store/migrate/migrations/0128_maxgaspricegwei_vrf.sql

```diff
@@ -0,0 +1,7 @@
+-- +goose Up
+ALTER TABLE vrf_specs
+    ADD COLUMN "max_gas_price_gwei" BIGINT
+    CHECK (max_gas_price_gwei >= 0);
+
+-- +goose Down
+ALTER TABLE vrf_specs DROP COLUMN "max_gas_price_gwei";
```

Comment:
Curious about how this migration logic is used. are the goose up/goose down used for upgrades & downgrades here?

Reply:
Kind of, but we don't realistically downgrade in practice - it is just run as a sanity check during testing. i.e. there is no expectation that the downgrade restores any data, only the schema.

Reply:
Good note on data restoration. 

Somewhat unrelated, but I guess there's a clever(-er) way to do this migration, whereby we fill in the `max_gas_price_gwei` column with the key-specific gas price that is in the configuration tables already, that way it'll get "auto-filled" without having to set that field in the job-spec manually. I think that's better, but not sure if there are any hidden pitfalls there...